api/proto/teleport/accesslist/v1/accesslist.proto

```diff
@@ -84,6 +84,9 @@ message AccessListSpec {
 
   // owner_grants describes the access granted by owners to this access list.
   AccessListGrants owner_grants = 11;
+
+  // member_count is the number of members in the access list.
+  int32 member_count = 12;
```

Comment:
Can we consider using `google/protobuf/wrappers.proto` -> google.protobuf.Int32Value (optional init) and set member_count when needed instead of depending of on magic value like -1

Reply:
I think those wrappers are mostly for use with `Any`; `optional` fields are cheaper on the wire format and the codegen (with `protoc-gen-go`, which thankfully access lists use) is just a `*int32`, just like you'd write for an optional field in go.